### PR TITLE
fix baidu mail bug

### PR DESCRIPTION
**Pull request by caijunhui81**
# Proposed changes

Issue Number: close #xxx

## Problem Summary:

Describe the overview of changes.

## Checklist(Required)

1. Does it affect the original behavior: (Yes/No/I Don't know)
2. Has unit tests been added: (Yes/No/No Need)
3. Has document been added or modified: (Yes/No/No Need)
4. Does it need to update dependencies: (Yes/No)
5. Are there any changes that cannot be rolled back: (Yes/No)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: manager/doris-manager/config/webpack.dev.config.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ const path = require("path");
 const { merge } = require("webpack-merge");
 const { webpack } = require("webpack");
 
-const SERVER_QA = 'http://st01-meg-lijiu02-403.st01.baidu.com:8090';
+const SERVER_QA = 'http://127.0.0.1:8090';
 
 const SERVER_RD = 'http://www.baidu.com:8080';
```
Comment (wuyunfeng) [inline manager/doris-manager/config/webpack.dev.config.js:25]: 这里还有一个呢

**Review by wuyunfeng — CHANGES_REQUESTED** [review]

**Review by songchuangyuan — COMMENTED** [review]
+2

**Review by wuyunfeng — COMMENTED** [review]
LGTM

**Review by wuyunfeng — APPROVED** [review]
+1

**Pull request merged**